docs/settingUp.md

```diff
@@ -88,9 +88,14 @@ Eclipse IDE is our preferred development environment. Support requests related t
       ![devverification-guide-1.png](images/devverification-guide-1.png)
      
    * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
-     Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
-     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
-     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`
+   * Trailing whitespace: all forms of trailing whitespace is disallowed, blank lines included.
+   * Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
+     * Java:
+        * `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
+        * `Java → Editor → Save Actions → Additional actions → Configure → Remove trailing whitespace → All lines`
```

Comment:
It looks like we're opening a can of worms here.

They may be good practices, but I'm not very favourable for creating instructions at this level. For me this has crossed the level of micro-managing: "You should use IDEs configured this way, not any other.". The devs can configure these themselves if they want (e.g. my own Eclipse setting is much stricter than a freshly installed Eclipse).
For this reason I think we can skip the setting up instructions on removing trailing spaces. Even without it, removing trailing whitespace is not a difficult practice to follow.

Thoughts on this?